DrawerLayout.js

```diff
@@ -92,6 +92,11 @@ export default class DrawerLayout extends Component<PropType, StateType> {
     const touchX = new Animated.Value(0);
     const drawerTranslation = new Animated.Value(0);
 
+    // This is necessary to ensure that the value we get from stopAnimation is
+    // kept up to date. If we don't do this then values on the JS side never get
+    // updated and drawerTranslation value in JS will always be 0.
+    this.drawerTranslationListener = drawerTranslation.addListener(() => {});
```

Comment:
😢

Reply:
😢

Reply:
Are you sure we need start value to pass to `_animatedDrawer`? Looks like we use it to call `setValue` so maybe in some cases we can skip calling `setValue` and just let it animate

Reply:
yeah that would work too